spec/models/spree/shipment_spec.rb

```diff
@@ -55,21 +55,23 @@ def create_complete_order
                                        order: create_complete_order) }
       let!(:shipped) { create_shipment(state: 'shipped',
                                        order: create_complete_order) }
+      let!(:canceled) { create_shipment(state: 'canceled',
+                                        order: create_complete_order) }
 
       let(:query) { Spree::Shipment.exportable }
 
-      context 'given we require payment to ship' do
-        before { Spree::Config.require_payment_to_ship = true }
+      context 'given capture at notification is inactive' do
```

Comment:
I'd suggest rephrasing to: `context 'given capture at notification is false' do`